matrix_70X.txt

```diff
@@ -1,190 +1,190 @@
-RunDoubleMuon2015Bwf134p703 134.703_RunDoubleMuon2015B+RunDoubleMuon2015B+HLTDR2_50ns+RECODR2_50nsreHLT+HARVESTDR2_50nsreHLT/step3.root reRECO
-RunDoubleMuon2015Bwf134p703 134.703_RunDoubleMuon2015B+RunDoubleMuon2015B+HLTDR250ns+RECODR250nsreHLT+HARVESTDR250nsreHLT/step3.root reRECO
-RunMET2015Bwf134p705 134.705_RunMET2015B+RunMET2015B+HLTDR250ns+RECODR250nsreHLT+HARVESTDR2reHLT/step3.root reRECO
-RunMET2015Bwf134p705 134.705_RunMET2015B+RunMET2015B+HLTDR250ns+RECODR250nsreHLT+HARVESTDR250nsreHLT/step3.root reRECO
-RunMET2015Bwf134p705 134.705_RunMET2015B+RunMET2015B+HLTDR2_50ns+RECODR2_50nsreHLT+HARVESTDR2_50nsreHLT/step3.root reRECO
-RunDoubleEG2015Cwf134p802 134.802_RunDoubleEG2015C+RunDoubleEG2015C+HLTDR225ns+RECODR225nsreHLT+HARVESTDR2reHLT/step3.root reRECO
-RunDoubleEG2015Cwf134p802 134.802_RunDoubleEG2015C+RunDoubleEG2015C+HLTDR225ns+RECODR225nsreHLT+HARVESTDR225nsreHLT/step3.root reRECO
-RunDoubleEG2015Cwf134p802 134.802_RunDoubleEG2015C+RunDoubleEG2015C+HLTDR2_25ns+RECODR2_25nsreHLT+HARVESTDR2_25nsreHLT/step3.root reRECO
-RunDoubleMuon2015Cwf134p803 134.803_RunDoubleMuon2015C+RunDoubleMuon2015C+HLTDR225ns+RECODR225nsreHLT+HARVESTDR2reHLT/step3.root reRECO
-RunDoubleMuon2015Cwf134p803 134.803_RunDoubleMuon2015C+RunDoubleMuon2015C+HLTDR225ns+RECODR225nsreHLT+HARVESTDR225nsreHLT/step3.root reRECO
-RunDoubleMuon2015Cwf134p803 134.803_RunDoubleMuon2015C+RunDoubleMuon2015C+HLTDR2_25ns+RECODR2_25nsreHLT+HARVESTDR2_25nsreHLT/step3.root reRECO
-RunMET2015Cwf134p805 134.805_RunMET2015C+RunMET2015C+HLTDR225ns+RECODR225nsreHLT+HARVESTDR2reHLT/step3.root reRECO
-RunMET2015Cwf134p805 134.805_RunMET2015C+RunMET2015C+HLTDR225ns+RECODR225nsreHLT+HARVESTDR225nsreHLT/step3.root reRECO
-RunMET2015Cwf134p805 134.805_RunMET2015C+RunMET2015C+HLTDR2_25ns+RECODR2_25nsreHLT+HARVESTDR2_25nsreHLT/step3.root reRECO
-RunDoubleEG2015Dwf134p902 134.902_RunDoubleEG2015D+RunDoubleEG2015D+HLTDR2_25ns+RECODR2_25nsreHLT+HARVESTDR2_25nsreHLT/step3.root reRECO
-RunDoubleMuon2015Dwf134p903 134.903_RunDoubleMuon2015D+RunDoubleMuon2015D+HLTDR2_25ns+RECODR2_25nsreHLT+HARVESTDR2_25nsreHLT/step3.root reRECO
-RunMET2015Dwf134p905 134.905_RunMET2015D+RunMET2015D+HLTDR2_25ns+RECODR2_25nsreHLT+HARVESTDR2_25nsreHLT/step3.root reRECO
-RunSinglePh2015Dwf134p911 134.911_RunSinglePh2015D+RunSinglePh2015D+HLTDR2_25ns+RECODR2_25nsreHLT+HARVESTDR2_25nsreHLT/step3.root reRECO
-TTbar14TeV2017wf10039p0 10039.0_TTbar_14TeV+TTbar_Tauola_14TeV_2017_GenSimFullINPUT+DigiFull_2017+RecoFull_2017/step3.root RECO
-MinBias14TeV2017wf10046p0 10046.0_MinBias_TuneZ2star_14TeV+MinBias_TuneZ2star_14TeV_pythia6_2017_GenSimFullINPUT+DigiFull_2017+RecoFull_2017/step3.root RECO
-TenMuE2017wf10060p0 10060.0_TenMuE_0_200+TenMuE_0_200_pythia8_2017_GenSimFullINPUT+DigiFull_2017+RecoFull_2017/step3.root RECO
+RunMinBias2011Awf1000p0 1000.0_RunMinBias2011A+RunMinBias2011A+TIER0+SKIMD+HARVESTDfst2+ALCASPLIT step2.root RECO
```

Comment:
I'm changing my mind on formatting of this file.
Similar to the DQM.sh file, for most cases the miniAOD file can be detected automatically: it is (almost) always a separate file with the same stepX and it is of the form stepX_inMINIAOD\*.root (SIM may or may not be there).
So, let's go back.